### PR TITLE
use Correction instead of ExtendedCorrection to avoid having null

### PR DESCRIPTION
**Pull request by kmschoep-usgs**
attributes for corrections that break the report due to isEmptyOrBlank.

### CODE REVIEW

File: src/main/java/gov/usgs/aqcu/config/OAuth2ResourceServerConfig.java [hunk 1]
```diff
@@ -21,7 +21,8 @@ public void configure(HttpSecurity http) throws Exception {
 			.authorizeRequests()
 				.antMatchers("/", "/swagger-ui.html", "/swagger-resources/**", "/webjars/**", "/v2/**").permitAll()
 				.antMatchers("/health/**", "/info/**").permitAll()
-				.anyRequest().fullyAuthenticated()
+				//.anyRequest().fullyAuthenticated()
+				.anyRequest().permitAll()
```
Comment (zmoore-usgs) [inline src/main/java/gov/usgs/aqcu/config/OAuth2ResourceServerConfig.java]: Should switch this back to fullyAuthenticated

Comment (kmschoep-usgs) [reply src/main/java/gov/usgs/aqcu/config/OAuth2ResourceServerConfig.java]: whoops, yep!

**Comment by kmschoep-usgs** [comment]
thanks for the info, I'll do more digging if that's the case.  The problem I was running into was that when ExtendedCorrections was used, often domiantType was null, and the isEmptyOrBlank repgen function was filtering out those corrections completely.

**Comment by mhines-usgs** [comment]
re: corr & freehand: https://internal.cida.usgs.gov/jira/browse/AQCU-677

**Comment by kmschoep-usgs** [comment]
Yep, need to do the CopyPaste->Freehand conversion.

**Pull request merged**